Documentation/rust/docs.rst

```diff
@@ -35,6 +35,11 @@ target with the same invocation used for compilation, e.g.::
 
 	make LLVM=1 rustdoc
 
+To read the docs locally in your web browser, run e.g.::
+
+	firefox rust/doc/$CRATE/index.html
```

Comment:
```suggestion
	xdg-open rust/doc/kernel/index.html
```

`xdg-open` is more general. I think it is also simpler to tell them to open `kernel` so that they can copy-paste the command -- they can navigate to the other crates using the left-side panel which contains the list of crates.

Reply:
Sounds good